intel_extension_for_transformers/llm/runtime/graph/__init__.py

```diff
@@ -131,8 +131,11 @@ def init_from_bin(self, model_type, model_path, **generate_kwargs):
         self.model = self.module.Model()
         if "threads" not in generate_kwargs:
             threads = os.getenv("OMP_NUM_THREADS")
+            import platform
+            sys_platform = platform.platform().lower()
             if threads is None:
-                generate_kwargs["threads"] = len(os.sched_getaffinity(0))
+                if "windows" not in sys_platform:
```

Comment:
we should set threads num when on windows

Reply:
fixed